src/content/en/updates/2013/07/Alpha-transparency-in-Chrome-video.md

```diff
@@ -1,15 +1,16 @@
 project_path: /web/_project.yaml
 book_path: /web/updates/_book.yaml
-description: WebM just added alpha transparency, and support for it has landed in Chrome Canary.
+description: WebM just added alpha transparency, and support for it has landed in Chrome.
```

Comment:
Removing 'Canary' is good. Which version did it land in? Chrome 75. Please say so. Please make that change anywhere you've removed 'Canary'.

Reply:
Done. See https://github.com/google/WebFundamentals/pull/7986/commits/f38b1ffe8c7c1c21d793d311d0e1483492a1cc22